manual/development-process/test-accessibility.md

```diff
@@ -42,7 +42,7 @@ Navigate through the page using a screen reader. Choose one which works on your
 operating system:
 
 * [VoiceOver] for MacOS (Built-in) - Safari should be used when testing with VoiceOver (rather than Chrome/Firefox etc).
-* [NVDA] for Windows
+* [Narrator] for Windows (Built-in) - An alternative would be [NVDA]
```

Comment:
I think we should check this with Sarra and Pauline, if you haven't already? The [design system recommend testing in NVDA and JAWS](https://design-system.service.gov.uk/accessibility/), rather than Narrator.

Reply:
Ah yes; true, I'll ask them. I tried Narrator and it provided a much better/smoother experience than NVDA, but I checked online and it seems to be a toss up as to which one is better and it seems NVDA might be more popular.

Reply:
I'm going to close this as after consulting with Pauline, I've become aware that Narrator is one of the lesser-used screen readers; sorry, I should have checked with Pauline and Sarra beforehand.